cmake/alpakaCommon.cmake

```diff
@@ -234,24 +235,27 @@ endif()
 
 #-------------------------------------------------------------------------------
 # Find OpenMP.
-if(ALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLE OR ALPAKA_ACC_CPU_B_SEQ_T_OMP2_ENABLE OR ALPAKA_ACC_CPU_BT_OMP4_ENABLE)
+if(ALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLE OR ALPAKA_ACC_CPU_B_SEQ_T_OMP2_ENABLE OR ALPAKA_ACC_ANY_BT_OMP5_ENABLE)
     find_package(OpenMP)
 
     if(OpenMP_CXX_FOUND)
-        if(ALPAKA_ACC_CPU_BT_OMP4_ENABLE)
-            if(OpenMP_CXX_VERSION VERSION_LESS 4.0)
-                message(FATAL_ERROR "OpenMP 4.0 is required but not supported!")
+         if(ALPAKA_ACC_ANY_BT_OMP5_ENABLED)
+           if(OpenMP_CXX_VERSION VERSION_LESS 4.0)
+               set(ALPAKA_ACC_ANY_BT_OMP5_ENABLE OFF CACHE BOOL "Enable the OpenMP 5.0 CPU block and thread back-end" FORCE)
```

Comment:
> The selection which jobs are required is part of repository settings. They are mentioned there by name.
> This is somehow fragile but it is what github offers at the moment. When we want to rename a job, we have to change the settings as well. I can do this, once this PR is ready or has been merged.

Ok. My rationale about changing this name is, that this is the only job with omp5 enabled, and will be one of only a few for a while. If for a future PR clang-10 fails while others succeed, it may be good to see, that it includes the omp5  backend.

Reply:
Done. Also added a warning for OpenMP < 5.0: https://github.com/alpaka-group/alpaka/pull/1126/commits/7fcec9209569693dfd5755a377581eb16663e5e4